R/index_nhdplus.R

```diff
@@ -229,6 +237,28 @@ get_flowline_index <- function(flines, points,
   return(matched)
 }
 
+check_search_radius <- function(search_radius, points) {
```

Comment:
I could see a reason for switching input parameters `function(points,search_radius)`

Reply:
But also see the reason for current order ... either way